### DISCUSSION TITLE
Where do we do the homework and how do we submit it.

### DISCUSSION THREAD

**Discussion by mim20017** [cooperrc/computational-mechanics#218] [Q&A]
I am a little confused on where we are supposed to answer the questions for the homework. Am I supposed to just edit the homework file in the github fork I made or type my answers in the imported copy on jupyter hub? Furthermore, am I just writing answers or am I creating code to allow me to solve the problems? And how do we submit our work to google classroom afterwards? I would appreciate a step by step walkthrough if possible.

**Comment by cooperrc** [comment]
Launch the HW in JupyterHub and run the code there. Then, you can create a print preview and submit that pdf to Classroom.
